hooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# pre-commit hook for git which builds the docker image for a given cloud-sdk version.
+# Copy into .git/hooks/  folder
```

Comment:
ok, -e added but the if any command in the sequence fails, the final exit status is what git looks for